From 61deb9efd404e11d084bd1cf30a5cfedda017496 Mon Sep 17 00:00:00 2001
From: tracierenea <tracie.perez@mavs.uta.edu>
Date: Tue, 5 Apr 2016 18:02:59 -0500
Subject: [PATCH 04/28] gr-fec: LDPC python function - addressing an attribute
 error

Minor fix to address this error:

Traceback (most recent call last):
  File "Generate_LDPC_matrix.py", line 78, in <module>
    parity_check_matrix.write_alist_file(alist_filename,bestH)
AttributeError: LDPC_matrix instance has no attribute 'write_alist_file'
---
 gr-fec/python/fec/LDPC/Generate_LDPC_matrix.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gr-fec/python/fec/LDPC/Generate_LDPC_matrix.py b/gr-fec/python/fec/LDPC/Generate_LDPC_matrix.py
index 696d957..a3862a6 100644
--- a/gr-fec/python/fec/LDPC/Generate_LDPC_matrix.py
+++ b/gr-fec/python/fec/LDPC/Generate_LDPC_matrix.py
@@ -75,5 +75,5 @@ print "\tgap  : %i" % g
 
 # Save the matrix to an alist file for future use:
 alist_filename = "n_%04i_k_%04i_gap_%02i.alist" % (n,k,g)
-parity_check_matrix.write_alist_file(alist_filename,bestH)
+write_alist_file(alist_filename,bestH)
 print '\nMatrix saved to alist file:', alist_filename, "\n"
-- 
2.1.4

